test/integration/cni/host_networking_test.go

```diff
@@ -36,13 +36,27 @@ const (
 	NEW_VETH_PREFIX            = "veth"
 	DEFAULT_MTU_VAL            = "9001"
 	DEFAULT_VETH_PREFIX        = "eni"
+	WARM_IP_TARGET             = "WARM_IP_TARGET"
+	DEFAULT_WARM_IP_TARGET     = "3"
```

Comment:
Can we move this to `pod_networking_suite_test.go` and use it in the `BeforeSuite` function too? 
In case the default needs to be updated in future.

Reply:
Putting it in `BeforeSuite` would change the values for all tests, and there are other tests that depend on the `WARM_IP_TARGET` being low. So I changed it so that it is set for just these tests and is reset after. It actually changes an additional (unnecessary) time, but that is because ginkgo doesn't have all "before all tests in this file only"/"after all tests in this file only" option

Reply:
My bad, I meant, let's move the `DEFAULT_WARM_IP_TARGET` declaration to the `pod_networking_suite_test.go` and use this here too https://github.com/aws/amazon-vpc-cni-k8s/pull/2105/files#diff-86c2f600945618a77023a9e8ecd685ddd43b0f4847fb9b3bb5955d61429aa667R91

I assume this is where the value `3` is coming from? So that in the future if this default value changes, it will be updated both in `BeforeSuite` and `BeforeEach`

Reply:
Ah got it, will do